### ISSUE TITLE
Scripts to reproduce the results in Figure 5 and Figure 8

### ISSUE THREAD

**Issue by Nan-S** [JJuOn/LFS-GAN#1]
Hi,
Thanks for your great work.
I am trying to reproduce the results in your paper Figure 5 and Figure 8 and do the comparison on other methods. Can I get the scripts to generate results in Figure 5 and Figure 8?
Thanks for your help!

**Comment by JJuOn** [comment]
Hi, sorry for the late response.

Unfortunately, we don't have the exact random noise vectors that have been used in our paper.
However, for the result of our framework and the pre-trained StyleGAN2, you can easily get some similar samples by running the `generate.py`.

If you fix the seed when generating images, you can obtain images of LFS-GAN and the pre-trained StyleGAN2 from the same random noise vector.

For the lifelong few-shot image generation task, since LFS-GAN is a modulation-based approach, we individually trained our network for each task.
The "After $\mathcal{T}_5$" means that our LFS-GAN is trained only on the task $\mathcal{T}_5$.
Therefore, if you want to generate images of "After $\mathcal{T}_5$", you can simply use our LFS-GAN trained on the "Babies" task.

I hope that my response may address your question.

**Comment by Nan-S** [comment]
Hi,

Thanks for your reply.
May I know which LDM model you are using in Table9 and Table10?

**Comment by JJuOn** [comment]
We used pre-trained checkpoint from the original implementation (https://github.com/CompVis/latent-diffusion#pretrained-ldms).

**Comment by Nan-S** [comment]
Thanks for the prompt reply. It helps a lot. I will proceed to close the issues.
